### PR TITLE
[CORE-11326] @goodhood/tracking: React 18 support

### PR DESCRIPTION
**Pull request by eliasbraun**
Nothing really to add here. Is it a non breaking change then?

### CODE REVIEW

File: packages/tracking/package.json [hunk 1]
```diff
@@ -28,10 +28,11 @@
     "lint:typescript": "../../scripts/check-typescript"
   },
   "devDependencies": {
-    "prop-types": "^15.8.1"
+    "prop-types": "^15.8.1",
+    "nebenan-helpers": "^6.0.0"
```
Comment (rejas) [inline packages/tracking/package.json:32]: why do we need this in devDependencies?

Comment (eliasbraun) [reply packages/tracking/package.json:32]: Added it here, because VSCode was shouting at me that this is missing in deps :)

Comment (rejas) [reply packages/tracking/package.json:32]: But its in the root package.json.... Webstorm doesnt complain :-(

Comment (eliasbraun) [reply packages/tracking/package.json:32]: Hmm true! Will get rid of it again!

File: packages/tracking/package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
   "bugs": {
     "url": "https://github.com/goodhood-eu/goodhood/issues"
   },
-  "version": "1.0.2",
+  "version": "2.0.0",
```
Comment (eliasbraun) [inline packages/tracking/package.json:14]: What do you think?

Comment (rejas) [reply packages/tracking/package.json:14]: Probably indeed a v2 since I remember peters mantra: Updated peerDependencies === Updated Major Version :-)

Comment (peterhass) [reply packages/tracking/package.json:14]: This PR is **adding** support for react 18, so it's not a breaking change -> therefore it's 1.1.0

Comment (peterhass) [reply packages/tracking/package.json:14]: Wrote a small section on the wiki about it 

https://wiki.nebenan.de/display/AWSM/1st+Party+Packages#id-1stPartyPackages-Situation2.1:Addingsupportformoreversions

**Review by peterhass — CHANGES_REQUESTED** [review]
-> minor version bump

**Review by rejas — APPROVED** [review]

**Review by peterhass — APPROVED** [review]
Looks good but how did you test it if our preview is still running react 16?

https://github.com/goodhood-eu/goodhood/blob/master/package.json#L70

**Comment by eliasbraun** [comment]
Already solved in
https://github.com/goodhood-eu/goodhood/pull/198

**Pull request closed without merging**